conda_rpms/build_rpm_structure.py

```diff
@@ -163,21 +171,12 @@ def create_rpmbuild_content(repo, target, config):
             #--------------- New for this ---------
 
             # Keep track of the labels which have tags - its those we want.
-            tags = []
-            envs = []
             for label, tag in labelled_tags.items():
-                deployed_name = tag.split('-', 2)[2]
```

Comment:
Looks like lines 169-171 are unused. Then looks like you're adding to the tags and envs list with each iteration only to loop through those lists again to call create_rpmbuild_for_tag. I think I've just tidied it up - have I deleted something important?